18_regexp_lab/01.py

```diff
@@ -1,12 +1,11 @@
-"""
-Write a program that reads data
-from property files.
-Each line in the file can either be:
-    An empty line
-    A comment line (Start with #)
-    A property line (of the form key = value)
-
-Write a program that takes a property file name and key
-as command line arguments and prints the requested value
-"""
-
+import sys
+import re
+
+cfg_file_path = sys.argv[1]
+key_in = sys.argv[2]
+with open(cfg_file_path, 'r') as cfg_file:
+    for line in cfg_file:
+        res = re.search(r'^\s*'+key_in+r'\s*=\s*(.*)$', line)

```

Comment:
Using external input as regexp is considered bad practice: The input may contain characters that have special meaning in regexp, such as `.`, `^`, `$`, etc. Better to read all values to a dictionary and search that.